evcouplings/visualize/pairs.py

```diff
@@ -1183,43 +1182,101 @@ def enrichment_pymol_script(enrichment_table, output_file,
         "cartoon putty"
     chain : str, optional (default: None)
         Use this PDB chain in residue selection
+    legacy: bool, optional (default: False)
+        Use legacy (2011) red and yellow colormap
+        for EC enrichment
     """
-    t = enrichment_table.query("enrichment > 1")
+    if legacy: 
+        t = enrichment_table.query("enrichment > 1")
+        t.loc[:, "b_factor"] = t.enrichment
+        # compute boundaries for highly coupled residues
+        # that will be specially highlighted
+        boundary1 = int(0.05 * len(t))  # top 5%
+        boundary2 = int(0.15 * len(t))  # top 15%
+
+        t.loc[:, "b_factor"] = t.enrichment
+
+        # set color for "low" enrichment (anything > 1)
+        t.loc[:, "color"] = "yelloworange"
 
-    # compute boundaries for highly coupled residues
-    # that will be specially highlighted
-    boundary1 = int(0.05 * len(t))  # top 5%
-    boundary2 = int(0.15 * len(t))  # top 15%
+        # high
+        t.loc[t.iloc[0:boundary1].index, "color"] = "red"
 
-    t.loc[:, "b_factor"] = t.enrichment
+        # medium
+        t.loc[t.iloc[boundary1:boundary2].index, "color"] = "orange"
+        
+    else:
+        t = deepcopy(enrichment_table)
+        t.loc[:, "b_factor"] = t.enrichment
+
+        # set boundaries for enrichment levels
+        # that will be specially highlighted
+        # create nine subsets
```

Comment:
Just out of curiosity - any particular reason for having 9 bins instead of 10?

Reply:
I used the ColorBrewer palettes, http://colorbrewer2.org/#type=sequential&scheme=BuPu&n=9, and the max number of colors available for the "BuPu" palette is 9 instead of 10